test/extended/templates/templateservicebroker_e2e.go

```diff
@@ -259,25 +280,78 @@ var _ = g.Describe("[Conformance][templates] templateservicebroker end-to-end te
 
 	deprovision := func() {
 		g.By("deprovisioning a service")
-		err := brokercli.Deprovision(context.Background(), cliUser, instanceID)
+		err := cli.TemplateClient().Template().Templates(cli.Namespace()).Delete(privatetemplate.Name, &metav1.DeleteOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
```

Comment:
what's the point of deleting the template here?

Reply:
What I've implemented is that the test explicitly checks that the namespace is left empty (inasmuch as this is possible).  We manually put that template in the namespace earlier (in the BeforeEach), therefore we must manually remove it for the test to succeed.